### ISSUE TITLE
The Mac OS Version/Files with high-res Textures(640x480) not working correct.

### ISSUE THREAD

**Issue by Xornado95** [BredaUniversityGames/DXX-Raytracer#4]
I have no sound/music and got artefacts too with the "improved" Descent Version. 

![scrn0000](https://github.com/BredaUniversityGames/DXX-Raytracer/assets/83348993/347b361d-b53c-45e1-87c9-fa946a03d009)

**Comment by dmcybe** [comment]
Mac version is way different than the dos version it uses cd readbook audio for music and mac source code was never released to public.

**Comment by Contingencyy** [comment]
I don't think this will ever get an official fix from us. Will still leave this open in case someone else deems this worth to look at and fix.
